deploy/controller/deployment.yaml

```diff
@@ -39,6 +39,19 @@ spec:
             requests:
               cpu: 200m
               memory: 200Mi
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
```

Comment:
Can you add them to controller's [helm charts](https://github.com/gocrane/helm-charts/blob/main/charts/scheduler/templates/controller-deployment.yaml) as well?

Reply:
ok